src/main/components/HeaderBar.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const HeaderBar = (props) => {
+  const {
+    header, authType, handleChange,
+  } = props;
+
+  return (
+    <div>
+      <p>Headers</p>
+      <form>
+      <select name='header' id='headerTypeInput' multiple={false} value={header}
```

Comment:
I think it makes more sense to just _label_ this as "Authentication", and move the “none” option to the `authType` select (since the header we send auth info on should never change)

Reply:
@conorsexton  I made the label to reflect Authentication. I am not following the second request in regards to "none" and authType